### ISSUE TITLE
[BUG] 0.3 Introduces CJS bug

### ISSUE THREAD

**Issue by wesbos** [opennextjs/opennextjs-cloudflare#198]
### Describe the bug

just updated to 0.3 and it seems one of the patches introduced a bug while importing a method on glob:

```bash
.../node_modules/.pnpm/@opennextjs+cloudflare@0.3.0_rimraf@3.0.2_wrangler@3.96.0/node_modules/@opennextjs/cloudflare/dist/cli/build/patches/to-investigate/inline-eval-manifest.js:2

import { globSync } from "glob";
         ^^^^^^^^
SyntaxError: Named export 'globSync' not found. The requested module 'glob' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'glob';
const { globSync } = pkg;
```

tested on node v23.4.0 and 22.11.0

### Steps to reproduce

happens when I run `pnpm run preview:worker`

Here are my scripts: 

```
    "build:worker": "opennextjs-cloudflare",
    "dev:worker": "wrangler dev --port 8771",
    "preview:worker": "pnpm run build:worker && pnpm run dev:worker",
    "deploy:worker": "pnpm run build:worker && wrangler deploy",
```

### Expected behavior

it does not error

### @opennextjs/cloudflare version

0.3.0

### Wrangler version

3.96.0

### next info output

```shell
Operating System:
  Platform: darwin
  Arch: arm64
  Version: Darwin Kernel Version 24.1.0: Thu Oct 10 21:03:15 PDT 2024; root:xnu-11215.41.3~2/RELEASE_ARM64_T6000
  Available memory (MB): 65536
  Available CPU cores: 10
Binaries:
  Node: 23.4.0
  npm: 10.9.2
  Yarn: 1.22.22
  pnpm: 9.10.0
Relevant Packages:
  next: 15.1.1 // There is a newer version (15.1.2) available, upgrade recommended! 
  eslint-config-next: 15.1.1
  react: 19.0.0
  react-dom: 19.0.0
  typescript: 5.7.2
Next.js Config:
  output: N/A
 ⚠ There is a newer version (15.1.2) available, upgrade recommended! 
   Please try the latest canary version (`npm install next@canary`) to confirm the issue still exists before creating a new issue.
   Read more - https://nextjs.org/docs/messages/opening-an-issue
```

**Comment by shadskii** [comment]
I just ran into this issue myself and it seems that pnpm was not resolving the correct version of `glob`. I was able to work around this by adding an override.
```
  "pnpm": {
    "overrides": {
      "glob": "^11.0.0"
    }
  }
```

Looking at the [source](https://github.com/opennextjs/opennextjs-cloudflare/blob/bbba0a8f2e6c911bd73f3245bb7ca893024d07c9/packages/cloudflare/package.json#L63) `glob` is a devDependency but should probably be a dependency instead.
